alma/lib/Utils/functions.php

```diff
@@ -150,15 +150,31 @@ function almaFormatPrice($cents, $id_currency = null)
 function getDateFormat($locale, $timestamp)
 {
     try {
+        if (!class_exists(IntlDateFormatter::class)) {
+            return getFrenchDateFormat($timestamp);
+        }
         $formatter = new IntlDateFormatter($locale, IntlDateFormatter::MEDIUM, IntlDateFormatter::NONE);
```

Comment:
this can not happen, `new IntelDateFormatter()` cannot "return" `null`

Reply:
Indead it can return null from de doc :) 
https://www.php.net/manual/fr/intldateformatter.create.php
`?[IntlDateFormatter]`